modular_skyrat/master_files/code/modules/client/preferences/headshot.dm

```diff
@@ -7,7 +7,7 @@
 	maximum_value_length = MAX_MESSAGE_LEN
 	/// Assoc list of ckeys and their link, used to cut down on chat spam
 	var/list/stored_link = list()
-	var/static/link_regex = regex("i.gyazo.com|media.discordapp.net|cdn.discordapp.com|a.l3n.co|b.l3n.co|c.l3n.co") //gyazo, discord, lensdump
+	var/static/link_regex = regex("i.gyazo.com|media.discordapp.net|cdn.discordapp.com|a.l3n.co|b.l3n.co|c.l3n.co|static.f-list.net/images/") //BUBBER EDIT: adds F-list's image host to the valid list
```

Comment:
You should still include the other hosts for clarity

Reply:
Updated the comment, should be better